iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Identifiable.java

```diff
@@ -69,6 +69,10 @@ default String getName() {
     default void addAlias(String alias) {
     }
 
+    default void addAlias(String alias, boolean ensureAliasUnicity) {
+        addAlias(alias);
+    }
+
```

Comment:
I wonder if it should be a property of the NetworkIndex (Network.setEnsureAliasUnicity) that can be set globally.

Reply:
I followed the same logic than `setEnsureIdUnicity` but that can be a good idea for both!

Reply:
Just a question, if I put:
``java
network.setEnsureUnicity(true);
network.getLoad("load").addAlias("alias", false);
```
What do you think the behavior should be? I'm not sure...

Reply:
I just came back to your comment, you seem to say it should not be set alias by alias but globally. I agree with you but do you think it should be the same for unique IDs?